sherlock/__init__.py

```diff
@@ -7,4 +7,6 @@
 
 __shortname__   = "Sherlock"
 __longname__    = "Sherlock: Find Usernames Across Social Networks"
-__version__     = "0.14.4"
+__version__     = "0.15.0"
+
+forgeApiLatestReleaseUrl = "https://api.github.com/repos/sherlock-project/sherlock/releases/latest"
```

Comment:
Since we use snake_case across the project, we should use snake_case here as well